### PR TITLE
Amplitude,Hacker earth event added

### PR DESCRIPTION
**Pull request by darthgera123**
event added for the month of november

### CODE REVIEW

File: static/data/2017/11/events.json [hunk 1]
```diff
@@ -107,4 +107,13 @@
     "website" : "https://www.eventbrite.com/e/48-hour-tech-launch-tickets-37738421635",
     "logo"  : "https://cdn.evbuc.com/eventlogos/1887522/1041361741-1.jpg" 
   }
```
Comment (geekyd) [inline static/data/2017/11/events.json]: @darthgera123 Please add a `,` here

**Comment by darthgera123** [comment]
ive made the changes but im not able to put up a pr even after going through the fix on the internet please help

File: static/data/2017/11/events.json [hunk 2]
```diff
@@ -106,5 +106,15 @@
     "type"  : "hackathon",
     "website" : "https://www.eventbrite.com/e/48-hour-tech-launch-tickets-37738421635",
     "logo"  : "https://cdn.evbuc.com/eventlogos/1887522/1041361741-1.jpg" 
+  },
+
```
Comment (geekyd) [inline static/data/2017/11/events.json]: Please remove extra spaces

**Comment by geekyd** [comment]
@darthgera123 Seems like there are conflicts, please resolve the conflicts without removing existing events

**Pull request closed without merging**